### DISCUSSION TITLE
Transparent widget

### DISCUSSION THREAD

**Discussion by aDioS786** [microsoft/XboxGameBarSamples#96] [Q&A]
Hi
I've set the widget to transparent and it has no controls, still when ran the background stays 5, it is not see through. It only gets see through if I pin the widget and then close the xbox game bar, is that how we can only achieve transparent widget?

Secondly, is there a way to set pinned to true/false through code rather clicking on it.

**Comment by vidager** [comment]
Hi aDioS786,

We are adding a feature in our April release of Game Bar that lets you declare you want transparency while game bar is in the foreground.  In most scenarios you shouldn't need transparency while Game Bar is in the foreground, but we understand some widgets want to do that to aid the user in positioning if the user needs to see what's behind.

We do not currently support programmatic pinning by design.  We're considering ways to allow this with proper user consent, but we want to ensure the user is in control of what widgets stay on their screen. 

Thanks,
Ben
